software/fishstick/src/gamecube/controller.rs

```diff
@@ -9,23 +13,18 @@ impl<J: Joybus> Controller<J> {
         Self { joybus }
     }
 
-    pub fn calibrate(&self) -> Result<(), ()> {
-        let send_packet = &[1, 2, 3];
-        let expected_receive = [4, 5, 6];
-
-        let receive_packet = self.joybus.send_receive(send_packet)?;
-
-        match receive_packet == expected_receive {
-            true => Ok(()),
-            false => Err(())
-        }
+    pub fn exists(&self) -> Result<bool, &'static str> {
```

Comment:
I presume the errors will become real `Error` types instead of strings in the future?

Reply:
yeah